tests/testthat/test-renderer-sixplot.R

```diff
@@ -0,0 +1,76 @@
+test_that("check rowspan, all plots in single row", {
+  plot_of_3_dots_data <- data.frame(x = c(1, 2, 3), y = c(1, 4, 9))
+  plot_of_2_dots_data <- data.frame(x = c(1, 2), y = c(1, 4))
```

Comment:
please avoid numbers in variable names, and use informative variable names instead.
please read https://docs.google.com/document/d/1W6-HdQLgHayOFXaQtscO5J5yf05G7E6KeXyiBJFcT7A which has coding guidelines in R.
This is a violation of:
-5 for variable names and/or identifiers containing numbers/digits where names or another programming technique could be used instead. solution: use informative (self-documenting) variable/identifier names or another programming technique.